### ISSUE TITLE
Why T5 do not generate the whole next sentence as one of the pretrain loss?

### ISSUE THREAD

**Issue by guotong1988** [huggingface/transformers#5388]
T5 has the next-sentence-predict loss. 
And the next-sentence-predict loss is not generating the whole next sentence but generating is_next or not_next token.

Thank you very much.

**Comment by patil-suraj** [comment]
Hi @guotong1988  what do you exactly mean by not generating the whole next sentence ?

**Comment by guotong1988** [comment]
Input the previous sentence and generate the next sentence.

**Comment by patrickvonplaten** [comment]
I don't really understand the question here. Are you asking why T5 was not trained with the objective to generate the complete next sentence?

**Comment by guotong1988** [comment]
Yes. I am asking why T5 was not pre-trained with the objective to generate the complete next sentence?

**Comment by guotong1988** [comment]
![image](https://user-images.githubusercontent.com/4702353/86194695-c5af5980-bb81-11ea-98d4-487355f8bb60.png)
https://github.com/google-research/text-to-text-transfer-transformer/issues/286
